mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -1,9 +1,22 @@
 CLASS net/minecraft/class_1890 net/minecraft/enchantment/EnchantmentHelper
 	METHOD method_17884 (Ljava/util/Map;Lnet/minecraft/class_2487;Lnet/minecraft/class_1887;)V
 		ARG 2 enchantment
-	METHOD method_22445 getEnchantments (Lnet/minecraft/class_2499;)Ljava/util/Map;
+	METHOD method_22445 fromTag (Lnet/minecraft/class_2499;)Ljava/util/Map;
+		COMMENT Loads enchantments from a list named binary tag.
 		ARG 0 tag
-	METHOD method_24365 getMatchingEnchantedEquipment (Lnet/minecraft/class_1887;Lnet/minecraft/class_1309;Ljava/util/function/Predicate;)Ljava/util/Map$Entry;
+	METHOD method_24365 chooseEquipmentWith (Lnet/minecraft/class_1887;Lnet/minecraft/class_1309;Ljava/util/function/Predicate;)Ljava/util/Map$Entry;
```

Comment:
```suggestion
	METHOD method_24365 findEquipmentWith (Lnet/minecraft/class_1887;Lnet/minecraft/class_1309;Ljava/util/function/Predicate;)Ljava/util/Map$Entry;
```
I think `find` is more appropriate than `choose` here.

Reply:
If you use "find", I'd call it `findAny` (see java stream), which... I like `choose` better

Reply:
`find` fails to indicate the random aspect of this method, while `choose` can have `null` result if none of the options can be chosen.

Reply:
If you want to look at java streams, you could call it findAny, it has basically the same behaviour.

Reply:
Java's stream `findAny` is more like entity selector's `sort=arbitrary`, while this method behaves like `sort=random`